{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T08:12:00.000+0000",
      "latitude": 51.2148281900345,
      "estimatedRestoration": "2022-04-12T22:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6215'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6215",
      "reference": "JR6215",
      "engineerOnSiteTime": "2022-04-12T09:48:00.000+0000",
      "name": "SP10 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP10 5AP",
        "SP10 5AR",
        "SP10 5NY"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.4577643558324753,
          51.2148281900345
        ],
        "type": "Point"
      },
      "UUID": "2e74788a-1a0d-46ac-8855-72dbb3524838",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.4577643558324753,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T15:01:00.000+0000",
      "latitude": 51.55125140132306,
      "estimatedRestoration": "2022-04-12T18:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6575'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6575",
      "reference": "JR6575",
      "name": "UB6 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB6 7QF",
        "UB6 7QG",
        "UB6 7QN",
        "UB6 7QU",
        "UB6 7QW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.3287532926713737,
          51.55125140132306
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-12T17:00:00.000+0000",
      "UUID": "92e68d32-f37e-4f16-b8ef-92827371de56",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.3287532926713737,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T14:57:00.000+0000",
      "latitude": 50.823882341602726,
      "estimatedRestoration": "2022-04-13T00:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6570'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6570",
      "reference": "JR6570",
      "engineerOnSiteTime": "2022-04-12T16:10:00.000+0000",
      "name": "PO19 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "PO19 8RS",
        "PO19 8RT",
        "PO19 8RX",
        "PO19 8SA"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.7846875417288116,
          50.823882341602726
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-12T16:15:00.000+0000",
      "UUID": "15d4d910-00a1-435f-abf1-7f98f4014985",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7846875417288116,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T13:17:00.000+0000",
      "latitude": 51.56065164180237,
      "estimatedRestoration": "2022-04-12T18:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6521'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6521",
      "reference": "JR6521",
      "engineerOnSiteTime": "2022-04-12T14:35:00.000+0000",
      "name": "RG9 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG9 3AX",
        "RG9 3AY",
        "RG9 3BA",
        "RG9 6RJ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.8708374821691891,
          51.56065164180237
        ],
        "type": "Point"
      },
      "UUID": "a64e3052-157c-4cb8-b9f9-cc49dcfcc6a9",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8708374821691891,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T16:45:00.000+0000",
      "latitude": 51.809769464415965,
      "estimatedRestoration": "2022-04-12T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6630'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6630",
      "reference": "JR6630",
      "name": "OX29 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "OX29 6PQ",
        "OX29 6PR"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.417920749130174,
          51.809769464415965
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-12T16:40:00.000+0000",
      "UUID": "3a688332-6c0f-401e-ad4a-8573b1017564",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.417920749130174,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T16:52:00.000+0000",
      "latitude": 51.19660486959456,
      "estimatedRestoration": "2022-04-12T18:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6476'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6476",
      "reference": "JR6476",
      "name": "BA12 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BA12 8DS",
        "BA12 8DT"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.1961301616674276,
          51.19660486959456
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-12T15:45:00.000+0000",
      "UUID": "0d37e1f7-adb2-4e18-86e2-5c1313c84757",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.1961301616674276,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T14:24:00.000+0000",
      "latitude": 51.51314853234712,
      "estimatedRestoration": "2022-04-12T20:15:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6558'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6558",
      "reference": "JR6558",
      "name": "SL1 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL1 2SH",
        "SL1 2SQ",
        "SL1 2SW",
        "SL1 7JP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.621086453821717,
          51.51314853234712
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-12T15:30:00.000+0000",
      "UUID": "2c64abd5-fee0-4514-b4d5-22b82af3fb03",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.621086453821717,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T11:40:00.000+0000",
      "latitude": 56.6665658305113,
      "estimatedRestoration": "2022-04-12T19:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6556'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6556",
      "reference": "JR6556",
      "name": "DD8 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "DD8 5AP",
        "DD8 5EG",
        "DD8 5HS",
        "DD8 5JZ",
        "DD8 5LG",
        "DD8 5LH",
        "DD8 5NL",
        "DD8 5PG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -3.0288648178352107,
          56.6665658305113
        ],
        "type": "Point"
      },
      "UUID": "79898b9e-80df-4af1-bed3-193582e9dfe0",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -3.0288648178352107,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T11:21:00.000+0000",
      "latitude": 56.69855679498512,
      "estimatedRestoration": "2022-04-12T19:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6431'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6431",
      "reference": "JR6431",
      "name": "PH36 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH36 4LH",
        "PH36 4LP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -6.104314185717279,
          56.69855679498512
        ],
        "type": "Point"
      },
      "UUID": "14c9246a-643b-426b-8552-2281e85c2c8c",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.104314185717279,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T17:02:00.000+0000",
      "latitude": 56.54281963330434,
      "estimatedRestoration": "2022-04-12T20:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6642'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6642",
      "reference": "JR6642",
      "name": "PH8 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PH8 0DW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -3.552694512757716,
          56.54281963330434
        ],
        "type": "Point"
      },
      "UUID": "45bddb27-02e0-4aa0-a667-f22a0c338994",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -3.552694512757716,
      "resolved": false
    }
  ]
}
